api/signup.go

```diff
@@ -12,6 +12,33 @@ type SignupOrg struct {
 	Subdomain string `json:"subDomain"`
 }
 
+var reservedSubDomains = []string{
```

Comment:
@ssoroka also had a list of reserved subdomains. Perhaps you should work together on this

Reply:
I'm happy to add or edit this list! I should have mentioned that in the PR description. 

Please do suggest changes to this list and I can update it.